### ISSUE TITLE
.env file execution errors

### ISSUE THREAD

**Issue by grpandurangipwc** [google/exposure-notifications-verification-server#1752]
### TL;DR

While performing 21.1 deployment, when executed "source .env" after terraform apply, saw below error
gururaj_r_pandurangi@cloudshell:~/new_mvs_mirror/mvs-infrastructure-mirror/encv-staging$ source .env
-bash: export: `?': not a valid identifier
-bash: export: `secret://projects/640015588988/secrets/redis-auth/versions/1': not a valid identifier
-bash: export: `:': not a valid identifier
-bash: export: `': not a valid identifier
-bash: export: `?': not a valid identifier
-bash: export: `secret://projects/640015588988/secrets/redis-auth/versions/1': not a valid identifier
-bash: export: `:': not a valid identifier
-bash: export: `': not a valid identifier
gururaj_r_pandurangi@cloudshell:~/new_mvs_mirror/mvs-infrastructure-mirror/encv-staging$

**Expected behavior**

Execution of "source .env" should not error out

**Observed behavior**

Due to the below syntax, there was an error 
export CACHE_REDIS_PASSWORD=var.redis_enable_auth ? "secret://projects/640015588988/secrets/redis-auth/versions/1" : ""
export RATE_LIMIT_REDIS_PASSWORD=var.redis_enable_auth ? "secret://projects/640015588988/secrets/redis-auth/versions/1" : ""

### Reproduction

1.  executed "terraform plan"
2.  executed "terraform apply"
3.  executed "source .env"

**Screenshots**

**Environment**

- OS:
- Browser:
- Version:

**Additional information**
